examples/light-switch-app/nrfconnect/README.md

```diff
@@ -261,12 +261,18 @@ To enable matter CLI a light_switch-app example should be compiled with addition
 
     west build -b build-target -- -DCONFIG_CHIP_LIB_SHELL=y
 
-Use these commands to control lighting app via Matter CLI:
+Use these commands to control lighting device via Matter CLI:
```

Comment:
How about "device running lighting app"? Lighting device is a little bit confusing when we have light switch and light bulb.

Reply:
Hmm, you are right. I agree that it may be a bit confusing.